docs/configuration-reference/platforms/packet.md

```diff
@@ -201,7 +201,7 @@ node_type = var.custom_default_worker_type
 
 | Argument                              | Description                                                                                                                                                                                                                                                                       |     Default     |     Type     | Required |
 |---------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:---------------:|:------------:|:--------:|
-| `auth_token`                          | Packet Auth token. Use the `PACKET_AUTH_TOKEN` environment variable instead.                                                                                                                                                                                                      |        -        |    string    |  false   |
+| `auth_token`                          | Packet Auth token. Use the `PACKET_AUTH_TOKEN` environment variable instead. Needed for packet-ccm.                                                                                                                                                                               |        -        |    string    |   true   |
```

Comment:
This is not really _required_ from configuration point of view, as it can be supplemented using `PACKET_AUTH_TOKEN` environment variable. I would change it to:
```
Packet Project API token. Can be supplied using `PACKET_AUTH_TOKEN` environment variable.